conctr.agent.lib.nut

```diff
@@ -117,13 +148,19 @@ class Conctr {
 
 
     // 
-    // Sends data for persistance to Conctr
+    // Sends data to conctr
     // 
-    // @param  {Table or Array}          payload  - Table or Array containing data to be persisted
-    // @param  {Function (err,response)} callback - Callback function on http resp from Conctr
-    // @return {Null}
+    // @param  {Table or Array} payloadOrig - Table or Array containing data to be persisted
+    // @param  { {Function (err,response)} callback - Callback function on resp from Conctr through agent
     // 
-    function sendData(payload, callback = null) {
+    function sendData(payloadOrig, callback = null) {
+
+        local payload = {};
+
+        // Make a local copy to ensure that original payload is left unchanged
```

Comment:
This assumes it is a table and then checks later on whether it is a table or array (payload can never be an array). Also, payloadOrig is never used after this so I don't know why you are making a copy.